core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -467,6 +336,7 @@
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/PlayerConstants.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/AnswerClassificationService.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/AnswerFeedbackPairDirective.js"></script>
+  <
```

Comment:
Not fixed? https://github.com/oppia/oppia/pull/2844/files#diff-c8f7763b9f375dcc87525573f84622a9R339